selfdrive/updated.py

```diff
@@ -110,20 +110,18 @@ def set_consistent_flag():
 
 
 def set_update_available_params(new_version=False):
-  params = Params()
-
   t = datetime.datetime.utcnow().isoformat()
-  params.put("LastUpdateTime", t.encode('utf8'))
+  put_nonblocking("LastUpdateTime", t.encode('utf8'))
```

Comment:
Don't use asynchronous version here. There could be a race condition where the UpdateAvailableParam is set, but the release notes aren't there yet.

Updated is also no in a critical loop, so timeouts don't matter.